warden/app/app_config.go

```diff
@@ -203,134 +263,158 @@ var (
 		// We allow the following module accounts to receive funds:
 		// govtypes.ModuleName
 	}
+)
 
+func moduleConfig() depinject.Config {
```

Comment:
this was changed to a function instead of a var because vars run before `init`, and the `init` is the one that sets the sdk.Config with bech32 prefixes. we need these to be set so that module accounts have the correct prefix.